### DISCUSSION TITLE
Is there a model that extracts vocal harmonies only?

### DISCUSSION THREAD

**Discussion by WishyNova** [Anjok07/ultimatevocalremovergui#565] [Q&A]
Just curious if there was a model that isolated the harmonies of a song. Also, is there a guide on all the models that tells you their purpose?

**Comment by luca-brwn** [comment]
the karaoke model removes the background vocals from the lead (mdx is the cleanest result imo). but if you're talking about individual vocals within a harmony, there isn't a model for that yet. your best bet would be to use melodyne's polyphonic algorithim, or you can use hit n' mix. you can get a free trial for hit n mix if i am not mistaken
